perforce-workgroup/decisions/AYD-002_Unreal_deadline_job_submission.md

```diff
@@ -0,0 +1,56 @@
+# AYD-002 Unreal Deadline Job Submission
+
+- **Status:** Proposed
+- **Impact:**  Medium
+- **Contributors:** Ondrej Samohel <ondrej@ynput.io>
+- **Approved by:** -
+
+## Background
+
+AYON needs to create render and publishing job on Deadline
+using specified change list. Doing this from AYON has advantages like
+ability to run validators before job is processed on farm taking
+precious resources, etc. In case of perforce, validators would need to
+do some integrity checks probably.
+
+## Options
+
+### Option 1
+
+Create Deadline job based on render publishing instances created by AYON inside Unreal. Pass perforce change list to the job if perforce is available. 
+
+#### Pros
+ 
+	- Similar to current Deadline support in other DCC
+    - Provides feedback to artist about any error and automates issue resolution using Repair action
+    - Validates if everything is submitted to Perforce correctly before render job is created
+  
+#### Cons
+ 
+	- Takes control of the job creation so it hard to customize.
+    - Depends on the specific Deadline plugin needed for job processing on the farm
+	
+### Option 2
```

Comment:
i think this is the road we have to take.
we have the `UnrealPipelineBasic` plugin as a little reference